documentation/modules/exploit/windows/smb/smb_relay.md

```diff
@@ -224,14 +231,14 @@ One way to coax a user into creating an SMB connection is to embed it in a websi
 First, create the website (we're using Kali for this) with the following content:
 
 ```html
-<html>
-  <head>
-    <title>UNC Example</title>
-  </head>
-  <body>
-    <img src="file:////192.168.123.1/fake.jpg" width="0px" height="0px">
-  </body>
-</html>
+&lt;html&gt;
```

Comment:
I don't think this should be escaped? 👀 Normally the markdown renderer would handle this for us if it were embedded in an html site or such

Reply:
You're right it should be escaped automatically, but the renderer used by msfconsole does not do that. You can verify it from the before and after using `info -d`.

Reply:
Looks like we have a custom renderer for code blocks which wasn't escaping HTML correctly.

PR to fix this file up:
https://github.com/rapid7/metasploit-framework/pull/16509

Reply:
Just landed that and then backed out the docs changes here. Thanks for really fixing that.